arangod/Cluster/ClusterFeature.cpp

```diff
@@ -368,6 +368,9 @@ void ClusterFeature::prepare() {
 void ClusterFeature::start() {
   // return if cluster is disabled
   if (!_enableCluster) {
+    if (!_disableHeartbeat) {
```

Comment:
I think we removed `_disableHeartbeat`

Reply:
I know that I followed that flag around the source file and found it useless.  So maybe it was removed recently ... if not, I support that it should be removed.

Reply:
Yes there will be also some more conflicts with #5105, because I switched some stuff overt to `std::chrono`

Reply:
I like your std::chrono work.  Yes, it will be a pain for me in merge.  But such is life.  Kaveh wants my HeartbeatThread.{h, cpp} changes independently.  So you may have to merge that if it gets pushed before your code.